### ISSUE TITLE
can't go back to the home screen

### ISSUE THREAD

**Issue by chucksteinmn** [FrontendMasters/ios-issues#58]
* Many times I'm forced to exit the app manually because I can't go back to the home screen. If I swith to another app while a video is playing, and the FrontendMaster app becomes suspended/hibernated when I resume it then the < back button and the Hamburguer button are both unresponsive (click on them will just return to the video). In this case my only option is to exit the app forcefully.

https://app.intercom.com/a/apps/w1ukcwje/inbox/inbox/58955/conversations/12149500102129

**Comment by ecerulm** [comment]
I'm the original reporter of this issue, and I want to add that there is indeed a workaround:

Instead of clicking the "<" (Back button)  in the video, I discovered that I can **swipe right from the left border** to go to the Home screen. 


Still I would expect the "back button" to work. Or to have some other way to go back to Home screen (I didn't find the swipe right to be very intuitive, I found that I could swipe right by chance).
